### PR TITLE
Review

### PR DESCRIPTION
**Pull request by derekbekoe**

### CODE REVIEW

**Comment by funkeyfreak** [comment]
Thanks for creating this PR - I'll finally have time to review it this afternoon 😄

File: README.md [hunk 1]
```diff
@@ -1 +1,95 @@
-# convey
\ No newline at end of file
+# convey
+
+A command-line tool that makes sharing pipes between machines easy.
+
+## Install
+
```
Comment (funkeyfreak) [inline README.md:6]: I would recommend adding a go native install:

**Install using go**
```bash
go get github.com/derekbekoe/convey
convey -h
```

**Review by funkeyfreak — COMMENTED** [review]
Recomendation: code orginization

Typically, code orginization in a go file is recomended to be as follows:
```go
import (
      'fmt'
)

// these consts are located at the top of the "page"
// exported before non-exported
const (
     Exported="An example of an exported constant",
     notExported="An example of a non-expoted constant"
)

// vairables typically come after const - or can be sprinkled through out the code
// again, exported over non-exported
var ExportedString string

// make sure to document your vars
var nonExportedString string

// exported types are second after const OR third after vars - depends
// again, exported over non-exported
type TypeInterface interface {
     ExternalPrinter()     
     nonExternalPrintHelper(str string) (err error)
}

// comment can describe the struct
type ExportedType struct {
     ExportedProp nonExportedType
     nonExportedProp nonExportedType
}

// comment can descrive the string-type
type nonExportedType string

// "naked functions," that is functions which do not extend a type/implement an interface
// typoically go after the type decelarition
// there are exceptions (constructors, etc) but you ge the point
// again, exported before non-exported
func PrintExportedConst() {
     fmt.Printf("%v", Exported)
}

// make sure to document your functions
func printNonExportedConst() {
     fmt.Printf("%v", nonExported)
}

// now we get to the implementation of interfaces/extension of types
// again, exported functions before non-exported functions
func NewTypeInterface(exportedString, nonExportedString string) &TypeInterface {
     newExportedType := &ExportedType {
          ExportedProp: (nonExportedType)exportedString,
          nonExportedProp: (nonExportedType)nonExportedString
     }
}

// document the function
func (e *ExportedType) ExternalPrinter() {
 ...
}

// document the function
func (e *ExportedType) nonExternalPrintHelper(str string) (err error) {
 ...
}
```
see [format.go](https://golang.org/src/fmt/format.go) or, my personal favorite, [Printf](https://golang.org/src/fmt/print.go) as an example.

File: cmd/configure.go [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+	configureCmd.PersistentFlags().BoolVar(&forceWrite, "overwrite", false, "Overwrite current configuration")
+}
+
+var configureCmd = &cobra.Command{
+	Use:   "configure",
+	Short: "Configure Convey",
+	Run:   ConfigureCommandFunc,
+}
+
+// ConfigureCommandFunc is a handler for the configure command
```
Comment (funkeyfreak) [inline cmd/configure.go:31]: Suggestion: 

Lower-case comments - see [print.go](https://golang.org/src/fmt/print.go)

File: cmd/root.go [hunk 3]
```diff
@@ -15,37 +15,195 @@
...
 
 // Execute adds all child commands to the root command and sets flags appropriately.
 // This is called by main.main(). It only needs to happen once to the rootCmd.
 func Execute() {
 	if err := rootCmd.Execute(); err != nil {
-		fmt.Println(err)
-		os.Exit(1)
+		errorExit(err.Error())
 	}
 }
```
Comment (funkeyfreak) [inline cmd/root.go:209]: Typically, init and main are root-level functions 

```go
init(){
    ...
}

main(){
    ...
}
```

**Comment by funkeyfreak** [comment]
Great work - a few comments on style, otherwise doap!

File: cmd/root.go (hunk 3 above)
Comment (derekbekoe) [reply cmd/root.go:209]: It seems to conflict with the main() in main.go.

**Pull request closed without merging**